redis_rate_limit/__init__.py

```diff
@@ -140,8 +136,11 @@ def increment_usage(self, increment_by=1):
                              .format(increment_by=increment_by))
 
         try:
-            current_usage = self._redis.evalsha(
-                INCREMENT_SCRIPT_HASH, 1, self._rate_limit_key, self._expire, increment_by)
+            if self.client not in self.whitelisted_clients:
+                current_usage = self._redis.evalsha(
+                    INCREMENT_SCRIPT_HASH, 1, self._rate_limit_key, self._expire, increment_by)
+            else:
+                current_usage = 0
```

Comment:
I wouldn't touch this code.

Reply:
See my last comment and add this check at the top like if self.ignored_clients and self.client in self.ignored_clients: return 0

Reply:
Changes done as requested.